src/utils/check-pr/README.md

```diff
@@ -5,7 +5,7 @@
 * Python >= 3.7, and dependencies (see `requirements.txt`)
```

Comment:
another reference of functool v3 in contrib\deploy-onefuzz-via-azure-devops\deploy-onefuzz.yml needs to be updated